code/script.js

```diff
@@ -0,0 +1,183 @@
+                        reponame = repo.base.repo.name
+
+                        // storing the url for commits for each repo
+                        const commitsForRepos = repo.commits_url
+
+                        // Invoking the fetchCommits function
+                        fetchCommits(commitsForRepos, reponame)
+
+                        // storing the url for comments for each repo -> commented out because not used yet in code
+                        //const commentsForRepos = repo.comments_url
+
+                        // storing the pull request number for each repo
+                        const pullRequestNumber = repo.number
+
+                        // Invoking the fetchComments function
+                        fetchComments(pullRequestNumber, reponame)
+
+                        // storing the url for languages
+                        const usedLanguages = repo.base.repo.languages_url
+
+                        // Invoking the getLanguages function
+                        getLanguages(usedLanguages, reponame)
+
+                    })
+            })
+    })
+}
+                        
+// Function for fetching the commits for all repos
+const fetchCommits = (commitsForRepos, reponame) => {
+
+//-------------------------------FETCH 3------------------------------------------------
+    fetch(commitsForRepos, options)
+        .then(res => res.json())
+        .then(data => {
+
+        // storing the number of commits
+        const numberOfCommits = data.length
+
+        // printing info to the projectBoard with dynamic id
+        document.getElementById(`commit-${reponame}`).innerHTML += `
+        <p>Amount of commits: ${numberOfCommits}</p>
+        `
+            // looping through data for commit messages
+            data.forEach(item => {
+            // storing the commit message from each 
+                const commitMessages = item.commit.message
+                    
+            })     
+        })
+        .catch(err => console.error(err))
+}
+
+// Function for fetching the comments for each pull request
+const fetchComments = (pullRequestNumber, reponame) => {
+
+//-------------------------------FETCH 4------------------------------------------------
+    fetch(`https://api.github.com/repos/Technigo/${reponame}/issues/${pullRequestNumber}/comments`, options)
+        .then(res => res.json())
+        .then(data => {
+
+        // storing the review message -> commented out because not used yet in the code
+        //const reviewMessage = data[0].body
+
+        })    
+        .catch(err => console.error(err))
+}
+
+// Function for fetching the languages used in projects
+const getLanguages = (usedLanguages, reponame) => {
+    fetch(usedLanguages, options)
+        .then(res => res.json())
+        .then(data => {
+
+            // storing the info about used languages
+            const languages = Object.keys(data)
+        
+        })
```

Comment:
I can't see on your page that you are showing the languages used for the projects? Did you change your mind and didn't want to display the different languages or is it something that doesn't work?

Reply:
That is what happened, I came across a problem with showing all the languages (one from pull request not made by me was showing the info as null in the url). So I left it there, also for the reason that I would try to solve it later as an extra feature. But this could have been explained in the comments, so good pointing that out 👍